
2020-03-29 19:12:45 --- CRITICAL: Error [ 0 ]: Interface 'Generic' not found ~ APPPATH\classes\Model\Usuario.php [ 3 ] in D:\xampp\htdocs\aditum.local\system\classes\Kohana\Core.php:430
2020-03-29 19:12:45 --- DEBUG: #0 D:\xampp\htdocs\aditum.local\system\classes\Kohana\Core.php(430): require()
#1 [internal function]: Kohana_Core::auto_load('Model_Usuario')
#2 D:\xampp\htdocs\aditum.local\application\classes\Controller\Usuario.php(16): spl_autoload_call('Model_Usuario')
#3 D:\xampp\htdocs\aditum.local\system\classes\Kohana\Controller.php(84): Controller_Usuario->action_novo()
#4 [internal function]: Kohana_Controller->execute()
#5 D:\xampp\htdocs\aditum.local\system\classes\Kohana\Request\Client\Internal.php(94): ReflectionMethod->invoke(Object(Controller_Usuario))
#6 D:\xampp\htdocs\aditum.local\system\classes\Kohana\Request\Client.php(114): Kohana_Request_Client_Internal->execute_request(Object(Request), Object(Response))
#7 D:\xampp\htdocs\aditum.local\system\classes\Kohana\Request.php(1000): Kohana_Request_Client->execute(Object(Request))
#8 D:\xampp\htdocs\aditum.local\public\index.php(118): Kohana_Request->execute()
#9 {main} in D:\xampp\htdocs\aditum.local\system\classes\Kohana\Core.php:430
2020-03-29 19:14:07 --- CRITICAL: Error [ 0 ]: Interface 'Model_Generic' not found ~ APPPATH\classes\Model\Usuario.php [ 3 ] in D:\xampp\htdocs\aditum.local\system\classes\Kohana\Core.php:430
2020-03-29 19:14:07 --- DEBUG: #0 D:\xampp\htdocs\aditum.local\system\classes\Kohana\Core.php(430): require()
#1 [internal function]: Kohana_Core::auto_load('Model_Usuario')
#2 D:\xampp\htdocs\aditum.local\application\classes\Controller\Usuario.php(16): spl_autoload_call('Model_Usuario')
#3 D:\xampp\htdocs\aditum.local\system\classes\Kohana\Controller.php(84): Controller_Usuario->action_novo()
#4 [internal function]: Kohana_Controller->execute()
#5 D:\xampp\htdocs\aditum.local\system\classes\Kohana\Request\Client\Internal.php(94): ReflectionMethod->invoke(Object(Controller_Usuario))
#6 D:\xampp\htdocs\aditum.local\system\classes\Kohana\Request\Client.php(114): Kohana_Request_Client_Internal->execute_request(Object(Request), Object(Response))
#7 D:\xampp\htdocs\aditum.local\system\classes\Kohana\Request.php(1000): Kohana_Request_Client->execute(Object(Request))
#8 D:\xampp\htdocs\aditum.local\public\index.php(118): Kohana_Request->execute()
#9 {main} in D:\xampp\htdocs\aditum.local\system\classes\Kohana\Core.php:430
2020-03-29 19:14:57 --- CRITICAL: Error [ 0 ]: Using $this when not in object context ~ APPPATH\classes\Model\Usuario.php [ 36 ] in D:\xampp\htdocs\aditum.local\application\classes\Controller\Usuario.php:16
2020-03-29 19:14:57 --- DEBUG: #0 D:\xampp\htdocs\aditum.local\application\classes\Controller\Usuario.php(16): Model_Usuario::salvar(Array)
#1 D:\xampp\htdocs\aditum.local\system\classes\Kohana\Controller.php(84): Controller_Usuario->action_novo()
#2 [internal function]: Kohana_Controller->execute()
#3 D:\xampp\htdocs\aditum.local\system\classes\Kohana\Request\Client\Internal.php(94): ReflectionMethod->invoke(Object(Controller_Usuario))
#4 D:\xampp\htdocs\aditum.local\system\classes\Kohana\Request\Client.php(114): Kohana_Request_Client_Internal->execute_request(Object(Request), Object(Response))
#5 D:\xampp\htdocs\aditum.local\system\classes\Kohana\Request.php(1000): Kohana_Request_Client->execute(Object(Request))
#6 D:\xampp\htdocs\aditum.local\public\index.php(118): Kohana_Request->execute()
#7 {main} in D:\xampp\htdocs\aditum.local\application\classes\Controller\Usuario.php:16
2020-03-29 19:19:21 --- CRITICAL: Error [ 0 ]: Undefined class constant 'modelo' ~ APPPATH\classes\Model\Usuario.php [ 36 ] in D:\xampp\htdocs\aditum.local\application\classes\Controller\Usuario.php:16
2020-03-29 19:19:21 --- DEBUG: #0 D:\xampp\htdocs\aditum.local\application\classes\Controller\Usuario.php(16): Model_Usuario::salvar(Array)
#1 D:\xampp\htdocs\aditum.local\system\classes\Kohana\Controller.php(84): Controller_Usuario->action_novo()
#2 [internal function]: Kohana_Controller->execute()
#3 D:\xampp\htdocs\aditum.local\system\classes\Kohana\Request\Client\Internal.php(94): ReflectionMethod->invoke(Object(Controller_Usuario))
#4 D:\xampp\htdocs\aditum.local\system\classes\Kohana\Request\Client.php(114): Kohana_Request_Client_Internal->execute_request(Object(Request), Object(Response))
#5 D:\xampp\htdocs\aditum.local\system\classes\Kohana\Request.php(1000): Kohana_Request_Client->execute(Object(Request))
#6 D:\xampp\htdocs\aditum.local\public\index.php(118): Kohana_Request->execute()
#7 {main} in D:\xampp\htdocs\aditum.local\application\classes\Controller\Usuario.php:16
2020-03-29 19:19:50 --- CRITICAL: Error [ 0 ]: Undefined class constant 'modelo' ~ APPPATH\classes\Model\Usuario.php [ 36 ] in D:\xampp\htdocs\aditum.local\application\classes\Controller\Usuario.php:16
2020-03-29 19:19:50 --- DEBUG: #0 D:\xampp\htdocs\aditum.local\application\classes\Controller\Usuario.php(16): Model_Usuario::salvar(Array)
#1 D:\xampp\htdocs\aditum.local\system\classes\Kohana\Controller.php(84): Controller_Usuario->action_novo()
#2 [internal function]: Kohana_Controller->execute()
#3 D:\xampp\htdocs\aditum.local\system\classes\Kohana\Request\Client\Internal.php(94): ReflectionMethod->invoke(Object(Controller_Usuario))
#4 D:\xampp\htdocs\aditum.local\system\classes\Kohana\Request\Client.php(114): Kohana_Request_Client_Internal->execute_request(Object(Request), Object(Response))
#5 D:\xampp\htdocs\aditum.local\system\classes\Kohana\Request.php(1000): Kohana_Request_Client->execute(Object(Request))
#6 D:\xampp\htdocs\aditum.local\public\index.php(118): Kohana_Request->execute()
#7 {main} in D:\xampp\htdocs\aditum.local\application\classes\Controller\Usuario.php:16
2020-03-29 19:20:49 --- CRITICAL: Error [ 0 ]: Undefined class constant 'modelo' ~ APPPATH\classes\Model\Usuario.php [ 36 ] in D:\xampp\htdocs\aditum.local\application\classes\Controller\Usuario.php:16
2020-03-29 19:20:49 --- DEBUG: #0 D:\xampp\htdocs\aditum.local\application\classes\Controller\Usuario.php(16): Model_Usuario::salvar(Array)
#1 D:\xampp\htdocs\aditum.local\system\classes\Kohana\Controller.php(84): Controller_Usuario->action_novo()
#2 [internal function]: Kohana_Controller->execute()
#3 D:\xampp\htdocs\aditum.local\system\classes\Kohana\Request\Client\Internal.php(94): ReflectionMethod->invoke(Object(Controller_Usuario))
#4 D:\xampp\htdocs\aditum.local\system\classes\Kohana\Request\Client.php(114): Kohana_Request_Client_Internal->execute_request(Object(Request), Object(Response))
#5 D:\xampp\htdocs\aditum.local\system\classes\Kohana\Request.php(1000): Kohana_Request_Client->execute(Object(Request))
#6 D:\xampp\htdocs\aditum.local\public\index.php(118): Kohana_Request->execute()
#7 {main} in D:\xampp\htdocs\aditum.local\application\classes\Controller\Usuario.php:16
2020-03-29 19:23:15 --- CRITICAL: Kohana_Exception [ 0 ]: Database method list_columns is not supported by Kohana_Database_PDO ~ MODPATH\database\classes\Kohana\Database\PDO.php [ 241 ] in D:\xampp\htdocs\aditum.local\modules\orm\classes\Kohana\ORM.php:1847
2020-03-29 19:23:15 --- DEBUG: #0 D:\xampp\htdocs\aditum.local\modules\orm\classes\Kohana\ORM.php(1847): Kohana_Database_PDO->list_columns('usuarios')
#1 D:\xampp\htdocs\aditum.local\modules\orm\classes\Kohana\ORM.php(469): Kohana_ORM->list_columns()
#2 D:\xampp\htdocs\aditum.local\modules\orm\classes\Kohana\ORM.php(408): Kohana_ORM->reload_columns()
#3 D:\xampp\htdocs\aditum.local\modules\orm\classes\Kohana\ORM.php(266): Kohana_ORM->_initialize()
#4 D:\xampp\htdocs\aditum.local\modules\orm\classes\Kohana\ORM.php(46): Kohana_ORM->__construct(NULL)
#5 D:\xampp\htdocs\aditum.local\application\classes\Model\Usuario.php(34): Kohana_ORM::factory('Model_Usuario')
#6 D:\xampp\htdocs\aditum.local\application\classes\Controller\Usuario.php(16): Model_Usuario::salvar(Array)
#7 D:\xampp\htdocs\aditum.local\system\classes\Kohana\Controller.php(84): Controller_Usuario->action_novo()
#8 [internal function]: Kohana_Controller->execute()
#9 D:\xampp\htdocs\aditum.local\system\classes\Kohana\Request\Client\Internal.php(94): ReflectionMethod->invoke(Object(Controller_Usuario))
#10 D:\xampp\htdocs\aditum.local\system\classes\Kohana\Request\Client.php(114): Kohana_Request_Client_Internal->execute_request(Object(Request), Object(Response))
#11 D:\xampp\htdocs\aditum.local\system\classes\Kohana\Request.php(1000): Kohana_Request_Client->execute(Object(Request))
#12 D:\xampp\htdocs\aditum.local\public\index.php(118): Kohana_Request->execute()
#13 {main} in D:\xampp\htdocs\aditum.local\modules\orm\classes\Kohana\ORM.php:1847
2020-03-29 19:25:13 --- CRITICAL: Kohana_Exception [ 0 ]: Database method list_columns is not supported by Kohana_Database_PDO ~ MODPATH\database\classes\Kohana\Database\PDO.php [ 241 ] in D:\xampp\htdocs\aditum.local\modules\orm\classes\Kohana\ORM.php:1847
2020-03-29 19:25:13 --- DEBUG: #0 D:\xampp\htdocs\aditum.local\modules\orm\classes\Kohana\ORM.php(1847): Kohana_Database_PDO->list_columns('usuarios')
#1 D:\xampp\htdocs\aditum.local\modules\orm\classes\Kohana\ORM.php(469): Kohana_ORM->list_columns()
#2 D:\xampp\htdocs\aditum.local\modules\orm\classes\Kohana\ORM.php(408): Kohana_ORM->reload_columns()
#3 D:\xampp\htdocs\aditum.local\modules\orm\classes\Kohana\ORM.php(266): Kohana_ORM->_initialize()
#4 D:\xampp\htdocs\aditum.local\modules\orm\classes\Kohana\ORM.php(46): Kohana_ORM->__construct(NULL)
#5 D:\xampp\htdocs\aditum.local\application\classes\Model\Usuario.php(34): Kohana_ORM::factory('Model_Usuario')
#6 D:\xampp\htdocs\aditum.local\application\classes\Controller\Usuario.php(16): Model_Usuario::salvar(Array)
#7 D:\xampp\htdocs\aditum.local\system\classes\Kohana\Controller.php(84): Controller_Usuario->action_novo()
#8 [internal function]: Kohana_Controller->execute()
#9 D:\xampp\htdocs\aditum.local\system\classes\Kohana\Request\Client\Internal.php(94): ReflectionMethod->invoke(Object(Controller_Usuario))
#10 D:\xampp\htdocs\aditum.local\system\classes\Kohana\Request\Client.php(114): Kohana_Request_Client_Internal->execute_request(Object(Request), Object(Response))
#11 D:\xampp\htdocs\aditum.local\system\classes\Kohana\Request.php(1000): Kohana_Request_Client->execute(Object(Request))
#12 D:\xampp\htdocs\aditum.local\public\index.php(118): Kohana_Request->execute()
#13 {main} in D:\xampp\htdocs\aditum.local\modules\orm\classes\Kohana\ORM.php:1847
2020-03-29 19:31:12 --- CRITICAL: Kohana_Exception [ 0 ]: Database method list_columns is not supported by Kohana_Database_PDO ~ MODPATH\database\classes\Kohana\Database\PDO.php [ 241 ] in D:\xampp\htdocs\aditum.local\modules\orm\classes\Kohana\ORM.php:1847
2020-03-29 19:31:12 --- DEBUG: #0 D:\xampp\htdocs\aditum.local\modules\orm\classes\Kohana\ORM.php(1847): Kohana_Database_PDO->list_columns('usuarios')
#1 D:\xampp\htdocs\aditum.local\modules\orm\classes\Kohana\ORM.php(469): Kohana_ORM->list_columns()
#2 D:\xampp\htdocs\aditum.local\modules\orm\classes\Kohana\ORM.php(408): Kohana_ORM->reload_columns()
#3 D:\xampp\htdocs\aditum.local\modules\orm\classes\Kohana\ORM.php(266): Kohana_ORM->_initialize()
#4 D:\xampp\htdocs\aditum.local\modules\orm\classes\Kohana\ORM.php(46): Kohana_ORM->__construct(NULL)
#5 D:\xampp\htdocs\aditum.local\application\classes\Model\Usuario.php(34): Kohana_ORM::factory('Model_Usuario')
#6 D:\xampp\htdocs\aditum.local\application\classes\Controller\Usuario.php(16): Model_Usuario::salvar(Array)
#7 D:\xampp\htdocs\aditum.local\system\classes\Kohana\Controller.php(84): Controller_Usuario->action_novo()
#8 [internal function]: Kohana_Controller->execute()
#9 D:\xampp\htdocs\aditum.local\system\classes\Kohana\Request\Client\Internal.php(94): ReflectionMethod->invoke(Object(Controller_Usuario))
#10 D:\xampp\htdocs\aditum.local\system\classes\Kohana\Request\Client.php(114): Kohana_Request_Client_Internal->execute_request(Object(Request), Object(Response))
#11 D:\xampp\htdocs\aditum.local\system\classes\Kohana\Request.php(1000): Kohana_Request_Client->execute(Object(Request))
#12 D:\xampp\htdocs\aditum.local\public\index.php(118): Kohana_Request->execute()
#13 {main} in D:\xampp\htdocs\aditum.local\modules\orm\classes\Kohana\ORM.php:1847
2020-03-29 19:35:46 --- CRITICAL: Kohana_Exception [ 0 ]: Database method list_columns is not supported by Kohana_Database_PDO ~ MODPATH\database\classes\Kohana\Database\PDO.php [ 241 ] in D:\xampp\htdocs\aditum.local\modules\orm\classes\Kohana\ORM.php:1847
2020-03-29 19:35:46 --- DEBUG: #0 D:\xampp\htdocs\aditum.local\modules\orm\classes\Kohana\ORM.php(1847): Kohana_Database_PDO->list_columns('usuarios')
#1 D:\xampp\htdocs\aditum.local\modules\orm\classes\Kohana\ORM.php(469): Kohana_ORM->list_columns()
#2 D:\xampp\htdocs\aditum.local\modules\orm\classes\Kohana\ORM.php(408): Kohana_ORM->reload_columns()
#3 D:\xampp\htdocs\aditum.local\modules\orm\classes\Kohana\ORM.php(266): Kohana_ORM->_initialize()
#4 D:\xampp\htdocs\aditum.local\modules\orm\classes\Kohana\ORM.php(46): Kohana_ORM->__construct(NULL)
#5 D:\xampp\htdocs\aditum.local\application\classes\Model\Usuario.php(34): Kohana_ORM::factory('Model_Usuario')
#6 D:\xampp\htdocs\aditum.local\application\classes\Controller\Usuario.php(16): Model_Usuario::salvar(Array)
#7 D:\xampp\htdocs\aditum.local\system\classes\Kohana\Controller.php(84): Controller_Usuario->action_novo()
#8 [internal function]: Kohana_Controller->execute()
#9 D:\xampp\htdocs\aditum.local\system\classes\Kohana\Request\Client\Internal.php(94): ReflectionMethod->invoke(Object(Controller_Usuario))
#10 D:\xampp\htdocs\aditum.local\system\classes\Kohana\Request\Client.php(114): Kohana_Request_Client_Internal->execute_request(Object(Request), Object(Response))
#11 D:\xampp\htdocs\aditum.local\system\classes\Kohana\Request.php(1000): Kohana_Request_Client->execute(Object(Request))
#12 D:\xampp\htdocs\aditum.local\public\index.php(118): Kohana_Request->execute()
#13 {main} in D:\xampp\htdocs\aditum.local\modules\orm\classes\Kohana\ORM.php:1847